test/rootfs/setup.sh

```diff
@@ -148,7 +148,7 @@ chmod ug+s /usr/bin/tup
 sed 's/#user_allow_other/user_allow_other/' -i /etc/fuse.conf
 
 # install go
-curl -L https://s3.amazonaws.com/flynn-temp/godeb.tar.gz | tar xz
+curl -L j.mp/godeb | tar xz
```

Comment:
Can do, I believe this just redirects to the tarball on S3 anyways.
This was just a straight revert by you are right, makes sense to fix that up now.